client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -1084,3 +1084,11 @@
 .sites-dashboard.preview-hidden .notice-banner {
 	display: flex;
 }
+
+.sites-dashboard__notices {
+	padding: 16px;
+
+	@media ( min-width: $break-small ) {
+		padding: 16px 64px;
+	}
+}
```

Comment:
Similar case as above. We have a different padding set for this page. For the banners to align with the content, we are setting the padding here.